usaspending_api/awards/migrations/0097_transaction_views.py

```diff
@@ -9,80 +9,78 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('awards', '0096_removing_subaward_models'),
-        ('search', '0019_additional_transaction_search_fields'),
+        ("awards", "0096_removing_subaward_models"),
+        ("search", "0019_additional_transaction_search_fields"),
     ]
 
     operations = [
-        migrations.RunSQL(
-            sql=vw_transaction_fabs_sql,
-            reverse_sql="DROP VIEW IF EXISTS vw_transaction_fabs",
-            # Without this, Django will try to actually change the old table names in another migration
-            # This says we've already done it.
-            state_operations=[
-                migrations.AlterModelTable(
-                    name='transactionfabs',
-                    table='vw_transaction_fabs',
-                )
-            ]
-        ),
-        migrations.AlterModelOptions(
-            name='transactionfabs',
-            options={'managed': False},
-        ),
-
-        migrations.RunSQL(
-            sql=vw_transaction_fpds_sql,
-            reverse_sql="DROP VIEW IF EXISTS vw_transaction_fpds",
-            # Without this, Django will try to actually change the old table names in another migration
-            # This says we've already done it.
-            state_operations=[
-                migrations.AlterModelTable(
-                    name='transactionfpds',
-                    table='vw_transaction_fpds',
-                )
-            ]
-        ),
-        migrations.AlterModelOptions(
-            name='transactionfpds',
-            options={'managed': False},
-        ),
-
+        # migrations.RunSQL(
```

Comment:
# These two view operations were commented out because this migration and its already established
        # dependencies are not compatible with recent changes to the views.
        # The dependencies in this migration also couldn't be altered to reflect the new dependencies
        # because this migration has already ran before the new dependencies in prod.
        # For those reasons we have commented these operations out and moved them to a new migration
        # that implements the new dependency order.

Reply:
Can you add this as a comment in the actual file?

Reply:
Sure

Reply:
Done